packages/block-library/src/separator/edit.js

```diff
@@ -58,18 +54,31 @@ export default function SeparatorEdit( {
 
 	return (
 		<>
-			<InspectorControls group="advanced">
-				<HTMLElementControl
-					tagName={ tagName }
-					onChange={ ( value ) =>
-						setAttributes( { tagName: value } )
-					}
-					clientId={ clientId }
-					options={ [
-						{ label: __( 'Default (<hr>)' ), value: 'hr' },
-						{ label: '<div>', value: 'div' },
-					] }
-				/>
+			<InspectorControls>
+				<PanelBody title="Settings">
+					<SelectControl
+						label="HTML Element"
```

Comment:
```suggestion
						label={ __( 'HTML element' ) }
```

The label needs to be translatable. I don't have a strong opinion on casing, but this matches the previous label.

https://github.com/WordPress/gutenberg/blob/80a371ebe73755ab4b78ae8f9624b007c9c0e50d/packages/block-editor/src/components/html-element-control/index.js#L93

Reply:
Oh boy, that was a bad one to miss. Fixes have been pushed.